ee-feature-pack/galleon-shared/src/main/resources/modules/system/layers/base/io/github/resilience4j/core/main/module.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright The WildFly Authors
+  ~ SPDX-License-Identifier: Apache-2.0
+  -->
+<module xmlns="urn:jboss:module:1.9" name="io.github.resilience4j.core">
+
```

Comment:
jboss-api = private?

Reply:
Whoops. Yes, this module should be private.

Reply:
Fixed.